projects/plugins/boost/app/assets/src/js/pages/benefits/BenefitsInterstitial.svelte

```diff
@@ -73,10 +75,10 @@
 					this={PricingCard}
 					title={'Jetpack Boost'}
 					icon={`${ window.Jetpack_Boost.site.assetPath }../static/images/forward.svg`}
-					priceBefore={19.95}
-					priceAfter={9.95}
+					priceBefore={pricing.full_price / 12}
```

Comment:
Does the react component handle weird float values like 1.3333333 or `0.1 + 0.2`?

Reply:
I tested this with a few different values like 1.3333333, 0.1 + 0.2, '123', null, etc and the component handles it appropriately. Upon that, I think we should be sending predefined values to the UI anyway.

Reply:
Yeah, but `19.99 / 12 = 1.6658333333` so I' just wanted to make sure we're able to handle that :)